Annotate/OverlayView.swift

```diff
@@ -445,49 +448,65 @@ class OverlayView: NSView, NSTextFieldDelegate {
         return validPoints
     }
 
-    private func drawArrow(from start: NSPoint, to end: NSPoint, color: NSColor) {
+    private func drawArrow(from start: NSPoint, to end: NSPoint, color: NSColor, lineWidth: CGFloat) {
         let adaptedColor = adaptColorForBoard(color, boardType: currentBoardType)
 
-        let path = NSBezierPath()
-        path.move(to: start)
-        path.line(to: end)
-
-        // Calculate arrow head
-        let arrowLength: CGFloat = 25.0
-        let arrowAngle: CGFloat = .pi / 6
+        // Calculate arrow head dimensions for equilateral triangle
+        let sideLength: CGFloat = 25.0  // Length of each side of the equilateral triangle
```

Comment:
The arrowhead is currently a fixed size, which looks disproportionate at very thin or very thick line widths. Can we make the arrowhead size relative to the line width?